lib/kafka_rest/client.rb

```diff
@@ -49,7 +49,7 @@ def request(method, path, body: nil, content_type: nil)
           begin
             JSON.parse(response.body)
           rescue JSON::ParserError => e
-            raise KafkaRest::InvalidResponse, "Invalid JSON in response: #{e.message}"
+            raise KafkaRest::ResponseError, "Invalid JSON in response: #{e.message}"
```

Comment:
This doesn't work, the `KafkaRest::ResponseError` class has a constructor with 2 arguments.